### PR TITLE
Remove CNI, new IMDS and revendor.

### PR DESCRIPTION
**Pull request by alekar**
Remove CNI from titus-executor.

### CODE REVIEW

**Review by sargun — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
-# Mesos executor for Titus
+# Virtual-Kubelet executor for Titus
 
-Currently runs titus containers using Docker.
+[![Build status](https://badge.buildkite.com/378856785cd7805f1efad9b69086205f82ec69ac8ba18e9479.svg)](https://buildkite.com/netflix/titus-executor)
```
Comment (solarkennedy) [inline README.md]: Rebase maybe?

Comment (alekar) [reply README.md]: I merged master into this branch. I'm a little confused by this comment.

Comment (solarkennedy) [reply README.md]: Oh I see, sorry I didn't read and was looking at individual commits and not the overall diff

**Review by rgulewich — APPROVED** [review]

**Comment by alekar** [comment]
Tini removal makes it easier to create a new pull request.

**Pull request closed without merging**